job-server/src/spark.jobserver/WebApi.scala

```diff
@@ -139,6 +188,21 @@ class WebApi(system: ActorSystem,
   }
 
   /**
+   * Routes for listing, deletion of and storing data files
+   *    GET /data                     - lists all currently stored files
```

Comment:
yes, only the file names will not be 'file1' and 'file', but the full file path (in both cases, before and after the restart)